redbot/cogs/audio/core/commands/controller.py

```diff
@@ -617,8 +617,13 @@ async def command_stop(self, ctx: commands.Context):
     @commands.guild_only()
     @commands.cooldown(1, 15, commands.BucketType.guild)
     @commands.bot_has_permissions(embed_links=True)
-    async def command_summon(self, ctx: commands.Context, channel: Optional[discord.VoiceChannel] = None):
-        """Summon the bot to a voice channel."""
+    async def command_summon(
+        self, ctx: commands.Context, channel: Optional[discord.VoiceChannel] = None
+    ):
+        """Summon the bot to a voice channel.
+        You can specify a personalised voice room with a mention or an id. Otherwise, the bot will use the voice room you are in.
+        To be able to easily mention a voice room, you can use `#!` on the computer, as with `#` and text rooms.
```

Comment:
>         To be able to easily mention a voice room, you can use `#!` on the computer, as with `#` and text rooms.
This is technically wrong -

What if I'm using a computer but running the discord android app which doesn't have support for this.




IMO -  your docstring should be changed to the following - I would make a suggestion but Github isn't letting me.

```
"""Summon the bot to a voice channel.

        You can specify a personalised voice room with an id. Otherwise, the bot will use the voice room you are in.

        To learn how to get a channel ID please read the following help article <https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID>
```

In other words, the `#!` functionality you outline shouldn't be mentioned until it is widely supported across all official discord clients - otherwise, it will just confuse users.